modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/data/provider/test/DDMRESTDataProviderTest.java

```diff
@@ -583,6 +587,10 @@ private DDMDataProviderRequest _createDDMDataProviderRequest(
 			builder.withParameter("filterParameterValue", filterParameterValue);
 		}
 
+		if (locale != null) {
+			builder.withLocale(locale);
+		}
+
 		if (Validator.isNotNull(inputParameterName)) {
 			builder.withParameter(
 				inputParameterName, GetterUtil.getString(inputParameterValue));
```

Comment:
@p-albuquerque do we really need to do all this `null` checking? Please check if we can change it to:

```
private DDMDataProviderRequest _createDDMDataProviderRequest(
	long ddmDataProviderId, String filterParameterValue, Locale locale,
	String inputParameterName, String inputParameterValue,
	String paginationEnd, String paginationStart) {

	return DDMDataProviderRequest.Builder.newBuilder(
	).withDDMDataProviderId(
		String.valueOf(ddmDataProviderId)
	).withLocale(
		locale
	).withParameter(
		inputParameterName, inputParameterValue
	).withParameter(
		"filterParameterValue", filterParameterValue
	).withParameter(
		"paginationEnd", paginationEnd
	).withParameter(
		"paginationStart", paginationStart
	).build();
}
```